src/shared/utils/networks.ts

```diff
@@ -36,7 +36,7 @@ const networks: Network[] = [
   },
   {
     network: 'dev',
-    entry: 's.devnet.ripple.test.net',
+    entry: 's.devnet.rippletest.net',
     port: 51235,
     unls: ['vl.devnet.rippletest.net'],
   },
```

Comment:
Is this hard coded value for the networks still used anywhere if you've switched to the table lookup?

Reply:
(If it's just the database setup file, maybe it should move into there?)